### ISSUE TITLE
Manually assign a conf to a renderer

### ISSUE THREAD

**Issue by Mik-S-UMS** [UniversalMediaServer/UniversalMediaServer#4685]
Someone on the forum asked how they can change the conf used for a device.

I was thinking it would be good if you could assign a conf to a device in the same way as you can assign a user in V14, instead of editing a conf to match that you think may be close enough.

This would make it easier to test existing confs on new devices, like for New Samsung TVs for example.

**Comment by SubJunk** [comment]
Yeah it's a good point. You probably already know, but there is a way to do it for all devices, with the "Default renderer" setting and the "Force default renderer" setting together.

This reminds me, I wonder if there is a better word than "renderer". People can get confused about it. I'm not sure if there is a better word that can mean both "device" and "app" though. Any ideas?

**Comment by Mik-S-UMS** [comment]
Yeah I knew about the force default but is not a good idea to use if you have many devices as it affects all of them.

I think there should also be something visual on the renderer icon to know that it not using the automatic renderer conf (maybe also showing what conf UMS thinks should be used if it was on automatic)

Yeah the word "renderer" does get a bit confused for some people but not sure of anything better as it can refer to both software and hardware. Maybe "media player" or something?

One thing I have thought about before is to have a device conf but also append a software conf if it is running an app, and this could be shown as the app's icon overlaid on the device picture. For example using VLC on a smart TV you would show a picture of the TV with a VLC logo on it, and in this case the VLC conf will have priority over the TV's conf. If the device an app is running on is not important for a conf then maybe you could just assign a custom picture for that device.

A similar thing could happen if you have a home theatre system that has better audio support than the TV that is connected to, like A Samsung TV that does not support DTS but a Yamaha AV receiver using audio pass-though does. If you could link the receiver's conf to the TV's conf then UMS will know not to transcode videos that use DTS. The only way to do this at the moment is to modify a conf as I have done this for several people on the forum with similar setups.

**Comment by SubJunk** [comment]
Oh yeah I like "media player". It is more descriptive than "renderer". What do you think @UniversalMediaServer/developers ?
